src/Dropdown/Dropdown.tsx

```diff
@@ -94,17 +99,13 @@ export const Dropdown = forwardRef(function Dropdown(
       <DropdownContainer>
         <StyledSelect
           id={id}
-          defaultValue={
-            list.length === 1
-              ? String(list[0].value)
-              : defaultValue
-              ? defaultValue
-              : placeholder
-          }
           disabled={disabled || list.length < 1}
-          onChange={(e: FormEvent<HTMLSelectElement>) => {
-            onSelect && onSelect(e.currentTarget.value)
-            onInputChange && onInputChange(e)
+          onChange={(event) => {
+            const value = event.currentTarget.value
+
+            onSelect?.(value)
+            onInputChange?.(event)
```

Comment:
Not sure a second event is needed tbh, is it used?

Reply:
yeah I think FNOL added it cause `onSelect` return the string value while they needed the actual event

Reply:
`onInputChange` is not used at all throughout FNOL. It was added before the correct use of the input components & react-hook-forms was established and can be removed now